content/get_start/UseCase_Workflows/revit.md

```diff
@@ -0,0 +1,108 @@
+
+
+ ##### **2. Define assemblies, internal loads, HVAC**
+- Envelope Construction: Assign envelope assemblies using either: 
+    - Generic layer-by-layer assemblies from Revit built-in library (typical for SD phase). 
+    - Schematic construction types present in the Revit build-in library (typical for DD phase).
+    - Actual construction defined in the Revit model (typical for CD phase).
+
+- Internal Loads: Assign internal loads based on the default loads present in Revit. The defaults could be based on the building typology (preferred for SD phase) or on space by space method (preferred for DD phase) according to various standards and databases (e.g., ASHRAE 90.1, ASHRAE 62.1 and CBECS data). For a CD phase, additionally, the lighting definitions are based on the actual modeled luminaires. 
+
+- Thermal Zones: Assign the thermal zones. It could be single thermal zone for each floor of the building for an SD stage model or based on the core/perimeter layout for DD and CD stage models. A single thermal zone may encompass several spaces. 
+
+- HVAC: Assign an HVAC system. 
+    - For an SD model, HVAC system could be skipped since an ideal load air system system automatically generated in the export process. 
+    - For DD and CD model, as an example for this workflow, a variable air volume (VAV) reheat packaged rooftop unit can be modeled with one system serving each floor. Any appropriate HVAC system must be modeled. 
+
+
+ ##### **3. Export the model**
+Export the model from Revit (BIM) to OpenStudio (BEM) by any of the following methods. Few differences between the export methods are highlighted below. 
+- gbXML export: All gbxml exports include essential model information, such as geometry, construction, internal gain definitions and schedules. There are some differences mentioned for each below: 
+   - Revit native gbXML 
+   - Revit Systems Analysis gbXML : Additional information included related to HVAC objects. 
+   - Autodesk Insight gbXML : Additional information included related to HVAC objects.
+
+{{% notice info %}}
+Note that the basic import feature of the OpenStudio application (i.e. the user interface) only translates a limited set of gbXML contents including geometry, constructions, thermal zones, and schedules. The translation of the remaining contents can be achieved using OpenStudio measures.
+{{% /notice %}}
+
+- Revit Systems Analysis OpenStudio model 
+    - Offers a streamlined workflow 
+    - Effectively translates most of the gbXML content into BEM using OpenStudio measures. 
+    - Various export modes available like a) Use Conceptual Masses and Building Elements (can be preferred for SD models), b) Use Rooms or Spaces (can be preferred for DD and CD models)
```

Comment:
These modes of export are also available for the native gbxml export.

Reply:
I see, so if I get this correctly these are available only for native gbxml export and Revit Systems Analysis OpenStudio model? And not for Revit Systems Analysis gbXML and Autodesk Insight gbXML?

Reply:
I have restructured the narrative and now it focuses only on Revit Systems Analysis with a side note on gbxml export. So this comment is not there anymore and should not be required.